flixel/FlxSprite.hx

```diff
@@ -1207,6 +1225,7 @@ class FlxSprite extends FlxObject
 	
 	private function set_color(Color:FlxColor):Int
 	{
+		Color = 0x00FFFFFF & Color;
```

Comment:
It was from a week or so ago, let me remove and see if I notice any issues in my code. If not, I'll remove it.